sim/make/build.mk

```diff
@@ -0,0 +1,87 @@
+# See LICENSE for license details.
+
+################################################################################
+# Helper to run SBT in the project.
+################################################################################
+
+SBT_COMMAND ?= shell
+SBT_NON_THIN ?= $(subst $(SBT_CLIENT_FLAG),,$(SBT))
+.PHONY: sbt
+sbt:
+	cd $(base_dir) && $(SBT_NON_THIN) ";project $(firesim_sbt_project); $(SBT_COMMAND)"
+
+
+################################################################################
+# Target Configuration
+################################################################################
+
+firesim_source_dirs = \
+	$(addprefix $(firesim_base_dir)/,\
+			. \
+			midas \
+			midas/targetutils \
+			firesim-lib\
+	)
+firesim_main_srcs = $(foreach submodule, $(firesim_source_dirs),\
+	$(shell find $(submodule)/ -iname "[!.]*.scala" -print 2> /dev/null | grep -E 'src/main/scala'))
+firesim_test_srcs = $(foreach submodule, $(firesim_source_dirs),\
+	$(shell find $(submodule)/ -iname "[!.]*.scala" -print 2> /dev/null | grep -E 'src/test/scala'))
+
+TARGET_SBT_PROJECT ?= $(firesim_sbt_project)
```

Comment:
Isn't this set in the `TARGET_PROJECT_MAKEFRAG` (e.g. `src/main/makefrag/firesim/config.mk`)? Why does this need to be potentially set again here?

Reply:
For projects that build tests, this defaults to the value I set here.